outputs.tf

```diff
@@ -17,3 +17,14 @@ output "kms_arn" {
   description = "AWS KMS key ID used to encrypt secrets, in case you wish to use it with other services."
   value       = aws_kms_key.key.arn
 }
+
+output "parameters_arn" {
+  description = "ARN for SSM secret names"
+  value       = aws_ssm_parameter.secure_param.arn
```

Comment:
secure_param is a `count` resource (creates multiple parameters) so I think your output would have to account for that
- you could point your code to the `additional-outputs` branch and try building to verify whether this output works